src/compounds/ItemGroup/ItemGroup.stories.jsx

```diff
@@ -9,18 +9,44 @@ export default {
 
 const Template = (args) => <ItemGroup {...args} />
 
-export const Default = Template.bind({})
-Default.args = {
+export const Vertical = Template.bind({})
+Vertical.args = {
```

Comment:
can we return the default option here? the intention for the default stories is to show people what they would get if they only passed in the required properties. from there, the other stories show 1+ variations to the default component by including other available props.